src/ucs/config/parser.c

```diff
@@ -379,6 +398,111 @@ int ucs_config_sprintf_time(char *buf, size_t max, void *src, const void *arg)
     return 1;
 }
 
+int ucs_config_sscanf_bw(const char *buf, void *dest, const void *arg)
+{
+    double *dst     = (double*)dest;
+    char    str[16] = {0};
+    int     offset  = 0;
+    size_t  divider;
+    double  units;
+    double  value;
+    int     num_fields;
+
+    num_fields = sscanf(buf, "%lf%16s", &value, str);
+    if (num_fields < 2) {
+        return 0; /* bytes or bits? */
```

Comment:
this comment is not very clear.. need to fix it or remove it

Reply:
comment is removed